### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by ssombra** [gdombiak/OctoPod#290]
This is a great app. I use it all time and the latest version is amazing. I see that somehow you are already getting info from the display layer progress plugin for current layer and height. would it be possible to also have the complete information the plugin displays in the LCD? I think it does it via M117 messages.

**Comment by gdombiak** [comment]
Hi @ssombra, thanks for the nice feedback! :)

Looking at the DisplayLayerProgress plugin I see that we could capture [what it displays on the LCD screen](https://github.com/OllisGit/OctoPrint-DisplayLayerProgress/blob/cd6895182112f5375c2e6de44e0fd8893ecf13d6/octoprint_DisplayLayerProgress/__init__.py#L377). Do you have an idea or suggestion on how/where to render this in OctoPod? 

I think that most people include in the LCD the same information that OctoPod renders in the main page (i.e. current layer / total layers) and maybe plus more info like estimated remaining time which OctoPod also renders. Wondering how to display this info without being potentially redundant but yet useful.

**Comment by ssombra** [comment]
That is true, for most people, the information already displayed in OctoPod is what they usually need (for me it is). I just thought that, as the plugin offers other kind of information, someone could eventually need it in specific situations. The options the plugin currently gives are:

```
[progress], [current_layer], [total_layers], [estimated_end_time], [printtime_left], [current_height], [total_height], [feedrate], [feedrate_g0], [feedrate_g1], [fanspeed], [last_layer_duration], [average_layer_duration], [changefilamenttime_left], [estimated_changefilament_time], [changefilament_count]

```

Maybe the app could have an additional tab in the panel module (currrently it has "porint job", temp graph, "custom controls" and "system commands") where it could display the info in the LCD.

**Comment by mneumark** [comment]
Similarly I have PrintTimeGenius and it does much better time calculations than the basic octoprint. It works especially for me as I have a Prusa MK3S which has the M73 gcode which the plugin reads. Would be nice to use the same estimate.

**Comment by gdombiak** [comment]
Hey @mneumark,

Have you tried using the "M73 ETA Override" plugin? I use that one instead of PrintTimeGenius. I see the exact end date on OctoPod, OctoPrint and on the printer's display. Same estimate that PrusaSlicer gave.
